benches/bench.rs

```diff
@@ -14,21 +14,13 @@ fn sigs_with_distinct_pubkeys<T: SigType>(
     })
 }
 
-fn sigs_with_same_pubkey<T: SigType>(
-) -> impl Iterator<Item = (VerificationKeyBytes<T>, Signature<T>)> {
-    let sk = SigningKey::<T>::new(thread_rng());
-    let vk_bytes = VerificationKey::from(&sk).into();
-    std::iter::repeat_with(move || {
-        let sig = sk.sign(thread_rng(), b"");
-        (vk_bytes, sig)
-    })
-}
-
 fn bench_batch_verify(c: &mut Criterion) {
     let mut group = c.benchmark_group("Batch Verification");
     for n in [8usize, 16, 24, 32, 40, 48, 56, 64].iter() {
```

Comment:
Doing this lets you clean up the `*`s throughout the rest of the function.

```suggestion
    for &n in [8usize, 16, 24, 32, 40, 48, 56, 64].iter() {
```